conf/documentation.conf

```diff
@@ -1059,6 +1045,87 @@ When enabled the /status page will only be available on the
 production network. By default this is disabled
 EOT
 
+[captive_portal.httpd_mod_qos]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Enables mod_qos for the portal
+EOT
+
+[captive_portal.httpd_mod_qos_maximum_connections_per_device]
+type=numeric
+options=enabled|disabled
+description=<<EOT
+The maximum connections per device
+EOT
+
+[captive_portal.httpd_mod_evasive]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Enables mod_evasive for the portal
+EOT
+
+[captive_portal.httpd_mod_evasive_white_list]
+type=text
```

Comment:
As @julsemaan says, not sure to understand that config parameter... also, it is not being used (not present in this PR except that for config files and translation file...)